mLRS/Common/dronecan/dronecan_generate_c_library.py

```diff
@@ -13,7 +13,22 @@
 import shutil
 import re
 import sys
+import venv
+import pathlib
+import subprocess
 
+script_folder = pathlib.Path(__file__).parent.resolve()
+venv_folder = os.path.join(script_folder, 'venv')
+venv.create(venv_folder, with_pip=True)
+python_path = os.path.join(venv_folder, "bin", "python")
```

Comment:
To resolve the issue of differents folder name containing python between Unix/MAC and Windows OS, same-ish code as in the original virtualenv commit ([here](https://github.com/pypa/virtualenv/commit/993ba1316a83b760370f5a3872b3f5ef4dd904c1)) can be used.
```suggestion
bin_dir = "bin"
if sys.platform == 'win32':
    bin_dir = "Scripts"
python_path = os.path.join(venv_folder, bin_dir, "python")
```